### PR TITLE
modify index.html

### PR DESCRIPTION
**Pull request by Daphineatim**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,18 +11,7 @@
...
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
```
Comment (Daphineatim) [inline index.html:25]: * Please do not leave commented code on your code base, it makes your code unclean.

File: index.html [hunk 2]
```diff
@@ -11,18 +11,7 @@
...
+   
     <div class="container">
       <header class="header">
         <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
```
Comment (Daphineatim) [inline index.html:17]: * Please consider using classes over id, id's are mostly used with elements that appear only once E.g. header, footer.

**Review by Daphineatim — COMMENTED** [review]
Hi @Daphineatim ,

Good work so far!
I can see you are almost there 👍
There are some issues that you still need to work on.

# Required Changes ♻️ 

_Check the comments under the review._

Cheers and Happy coding! 👏 👏 👏

**Review by keroloussamy — APPROVED** [review]
Hi @Daphineatim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Comments are polite.✔️
- Includes an explanation. ✔️
- Comments are clear and easy to understand. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**